bukkit/src/main/java/dev/aurelium/auraskills/bukkit/mana/ReadiedManaAbility.java

```diff
@@ -51,6 +55,29 @@ protected boolean materialMatches(String checked, Player player) {
         return false;
     }
 
+    protected void setHoldingMaterialDurability(Player player, int count, double multiplier) {
+        if (multiplier <= 0) return;
+
+        int damage = (int) (count * multiplier);
+        if (damage <= 0) return;
+
+        ItemStack item = player.getInventory().getItemInMainHand();
+        ItemMeta meta = Objects.requireNonNull(item.getItemMeta());
```

Comment:
You can remove the Objects.requireNonNull here because you have a null check on the next line